services/wallet/router.go

```diff
@@ -603,6 +647,11 @@ func (r *Router) suggestedRoutes(
 			continue
 		}
 
+		l1Fee, err := r.getL1Fee(ctx, network, addrTo, addrFrom, amountIn)
+		if err != nil && err != gaspriceoracle.ErrorNotAvailableOnChainID {
+			return nil, err
+		}
```

Comment:
Would it be possible to calculate this from fees.go and add the result to the struct here: https://github.com/status-im/status-go/blob/590e35272c8ea3682427a3f3a9df043f15539656/services/wallet/fees.go#L23-L31

Reason is that not every part use the send modal and they use this fees.go directly